api/agentpb/agent.proto

```diff
@@ -117,6 +117,20 @@ message StartActionRequest {
     string dsn = 1;
     string query = 2;
   }
+  // MySQLQueryShowParams describes MySQL SHOW query action parameters.
+  message MySQLQueryShowParams {
+    string dsn = 1;
+    string query = 2;
+  }
+  // PostgreSQLQueryShowParams describes PostgreSQL SHOW query action parameters.
+  message PostgreSQLQueryShowParams {
+    string dsn = 1;
+  }
```

Comment:
Why `PostgreSQLQueryShowParams` hasn't `query` field?

Reply:
It is always `SHOW ALL`.